platforms/platforms.bzl

```diff
@@ -3,12 +3,12 @@ load("@bazel_arm_toolchains//toolchains:toolchain_info.bzl", "ALL_CPU", "AVAILAB
 def all_platforms():
     for toolchain in AVAILABLE_TOOLCHAINS:
         for cpu in ALL_CPU:
-            name = "{}-{}-{}".format(cpu, toolchain["target_os"], toolchain["version"])
+            name = "{}-{}-{}-{}".format(toolchain["host_os"], cpu, toolchain["target_os"], toolchain["version"])
             native.platform(
                 name = name,
                 constraint_values = [
                     "//platforms/cpu:{}".format(cpu),
-                    "@platforms//os:{}".format(toolchain["target_os"]),
+                    "@platforms//os:{}".format(toolchain["target_os"]).replace("darwin","macos"),
```

Comment:
Can you make `os_constraint_name` a temp variable above with a comment of why this replace is needed?

Reply:
And below.

Reply:
done, let me know if you want to invert what @ograff and  I did here and use macos as the base instead of darwin

Reply:
@ograff authored this by the way